features/environment.py

```
@@ -0,0 +1,7 @@
+from allure_behave.hooks import allure_report
+
+
+def before_all(context):
```

Ruff (0.8.0)

Missing return type annotation for public function `before_all`

Add return type annotation: `None`

(ANN201)